resources/tickets.py

```diff
@@ -92,3 +92,19 @@ def post(self):
 
         ticket.save_to_db()
         return ticket.json(), 201
+
+    @pm_level_required
+    def delete(self):
+        self.parser.add_argument("ids", action="append")
+        data = Tickets.parser.parse_args()
+
+        if not ("ids" in data and type(data["ids"]) is list):
+            return {"message": "Ticket IDs missing in request"}, 400
+
+        for id in data["ids"]:
+            ticket = TicketModel.find(id)
+            if not ticket:
+                continue
+            ticket.delete_from_db()
```

Comment:
I have the same question from the previous PR #228 concerning how to handle id's that are non-existent. Do we ignore or do we abort the batch delete? I'm asking again, because it is still not clear to me based on the code changes alone.

Reply:
To me, it doesn't make sense to abort the deletion of a group of tickets if one of the tickets requested to delete is already gone (i.e. concurrent users deleting the same resources). The frontend just needs to fix the possible sync issue by re-requesting the resources or adjusting the displayed items based on the response of this call

Reply:
If that's the case then there are tests missing here to document this behavior. The logic here will abort on the first id that is non-existent in the db. Also, I believe a batch delete can be done in one go with something like this.
```
        TicketModel.query.filter(
            TicketModel.id.in_(data["ids"])
        ).delete(synchronize_session="fetch")
```

Reply:
Ah. I did not realize "find" threw a 404 if it did not get a match, while the function "find_by_id" I was using before was really what I wanted to use, but we are getting rid of that? I've replaced the find with the code you've given

Reply:
yeah, the code base was littered with find and return a 404 if it couldn't find it. So that has been abstracted away so that we're not repeating the same logic in every controller. "find_by_id" is removed because I don't think it was doing anything useful. There are sometimes cases where we want to look up an object by the id and not throw an error if it is not found, and that can be done easily with the `query.get(:id)` method.